scilpy/io/utils.py

```diff
@@ -494,3 +495,8 @@ def parser_color_type(arg):
         raise argparse.ArgumentTypeError(
             "Argument must be < " + str(MAX_VAL) + "and > " + str(MIN_VAL))
     return f
+
+
+def save_image(img_array, filename):
```

Comment:
Maybe we should add a comment saying `img_array[0, 0]` correspond to the bottom left corner of the image?

Reply:
I realize now that the name of the function might be too general and misleading as it might lead someone to use it without having an image that comes from `window.snapshot`. Which would potentially flip an image that doesn't need to be flipped. I'll adapt

Reply:
@CHrlS98 fixed in b2ce7f6